### PR TITLE
[Issue-60]: Translate function

### PR DESCRIPTION
**Pull request by ltuffery**
- Add `trans(key)` function
- Add `lang/` folder

### CODE REVIEW

File: srcs/django_code/etc/ssl/certs/cert.crt
Comment (swotex) [inline srcs/django_code/etc/ssl/certs/cert.crt:1]: Suppr le fichier

File: srcs/django_code/etc/ssl/certs/cert.key
Comment (swotex) [inline srcs/django_code/etc/ssl/certs/cert.key:1]: Suppr le fichier

File: srcs/django_code/front/js/main.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { Minichat } from "../components/minichat/minichat.js";
 import { Message } from "../components/minichat/message.js";
 import { Friend } from "../components/minichat/friend.js";
 import { ChatInput } from "../components/minichat/input.js";
+import { trans } from "./helpers.js";
```
Comment (swotex) [inline srcs/django_code/front/js/main.js:8]: not use

File: srcs/django_code/front/js/helpers.js
Comment (swotex) [inline srcs/django_code/front/js/helpers.js:1]: set a default language if cookie is wrong

**Review by swotex — CHANGES_REQUESTED** [review]

File: srcs/django_code/front/js/main.js (hunk 1 above)
Comment (swotex) [reply srcs/django_code/front/js/main.js:8]:
```suggestion

```

File: srcs/django_code/front/js/helpers.js
Comment (ltuffery) [reply srcs/django_code/front/js/helpers.js:1]: fix ici [107a78b](https://github.com/Rreyth/get-transcended/pull/61/commits/107a78b9b4740008433407bdf673aafba22facbf)

File: srcs/django_code/front/js/main.js (hunk 1 above)
Comment (ltuffery) [reply srcs/django_code/front/js/main.js]: fix ici [107a78b](https://github.com/Rreyth/get-transcended/pull/61/commits/107a78b9b4740008433407bdf673aafba22facbf)

File: srcs/django_code/etc/ssl/certs/cert.key
Comment (ltuffery) [reply srcs/django_code/etc/ssl/certs/cert.key:1]: fix ici [2490f5c](https://github.com/Rreyth/get-transcended/pull/61/commits/2490f5c2e4a4539f01696f2bd8535888df6aa55d)

File: srcs/django_code/etc/ssl/certs/cert.crt
Comment (ltuffery) [reply srcs/django_code/etc/ssl/certs/cert.crt:1]: fix ici [2490f5c](https://github.com/Rreyth/get-transcended/pull/61/commits/2490f5c2e4a4539f01696f2bd8535888df6aa55d)

File: srcs/django_code/front/js/helpers.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    const response = await fetch(`https://${location.hostname}:${location.port}/static/lang/${lang.value}.json`)
+
+    if (response.ok)
+    {
+        const value = (await response.json())[key]
+
+        if (value == null)
+        {
+            cookieStore.set({ name: "lang", value: default_lang })
+            return await translate(key)
```
Comment (swotex) [inline srcs/django_code/front/js/helpers.js:20]: Eviter les boucle infinie par un set par deffault

```suggestion
            value = "NULL"
```

**Review by swotex — CHANGES_REQUESTED** [review]
Pour les fichier json creer des sous categorie par page

File: srcs/django_code/front/js/helpers.js (hunk 2 above)
Comment (ltuffery) [reply srcs/django_code/front/js/helpers.js]: Fix ici [86c995b](https://github.com/Rreyth/get-transcended/pull/61/commits/86c995b51a72bdb4e9b062e397628af941ead2ee)

**Review by Rreyth — APPROVED** [review]
gogogo

**Review by swotex — APPROVED** [review]

**Pull request merged**